src/pageLayout/containers/TreeNav.tsx

```diff
@@ -22,12 +24,27 @@ import {SafeBlankLink} from 'src/utils/SafeBlankLink'
 
 // Types
 import {NavItem, NavSubItem} from 'src/pageLayout/constants/navigationHierarchy'
+import {AppState} from 'src/types'
 
-const TreeSidebar: FC = () => {
-  const org = useSelector(getOrg)
+import {showOverlay, dismissOverlay} from 'src/overlays/actions/overlays'
+
+type ReduxProps = ConnectedProps<typeof connector>
+
+const TreeSidebar: FC<ReduxProps & RouteComponentProps> = ({
+  showOverlay,
+  dismissOverlay,
+}) => {
   const {presentationMode, navbarMode, setNavbarMode} = useContext(
     AppSettingContext
   )
+  const userAccount = (state: AppState): boolean => {
+    const {quartzMe} = state.me
+    const accountType = quartzMe?.accountType ?? 'free'
+    return accountType !== 'free'
+  }
+  const isPayGCustomer = useSelector(userAccount)
```

Comment:
I think the naming and usage of this was a little bit confusing to me. It took me a second to realize that `userAccount` is a selector function. 

I think if we're going to take the time to write a selector, we should put it in a place where it's reusable. I'm not sure if this needs to be reusable yet though - there's a tension between the overhead of writing and importing a selector to be used in a single place and just using the data you already have access to in the component. I usually default to creating a selector if I have to access this data in multiple different components.

There are two options how to do this:
- if you want to keep it as a selector, put it in `me/selectors/index` and name it something that indicates it gets the payg status
- just set the `isPayGCustomer` (great variable name btw!) in this block of code without making a selector function

```ts
const accountType = state.me.quartzMe?.accountType ?? 'free'
const isPayGCustomer = accountType !== 'free'
```

also since there are subtle differences between `??` and `||`, if you haven't already, make sure that `??` is the operator you want, rather than `||` :)

Reply:
hmm.. looking back at it, I can see how `userAccount` can be a bit confusing to understand. I removed the selector function code. Setting the `PayGCustomer` like you mentioned is a much easier route.